### PR TITLE
[Tests] Introduce indexCI for npm test on CI

### PR DESCRIPTION
**Pull request by YongseopKim**
Let's introduce indexCI for npm test on CI - github action.

ONE-vscode-DCO-1.0-Signed-off-by: Yongseop Kim <yons.kim@samsung.com>

### CODE REVIEW

File: src/Tests/indexCI.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+/*
+ * This file comes from
+ * https://github.com/microsoft/vscode-extension-samples/blob/ba9a56e68277b6ba0ae3ee4593b6a864ba158b1f/helloworld-test-sample/src/test/suite/index.ts
+ */
+
+import {glob} from 'glob';
+import Mocha from 'mocha';
+import * as path from 'path';
+
+// TODO: if there is a way to pass from runTest to this file's run(), we could merge to one file.
```
Comment (YongseopKim) [inline src/Tests/indexCI.ts:48]: Please read this.

File: src/Tests/indexCI.ts [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+// NOTE: indexCI.ts is copied from index.ts. On CI, some tests have limitations so they have to be excluded.
+// This function does it.
+export function run(): Promise<void> {
+  const mocha = new Mocha({ui: 'tdd', color: true, fgrep: '@Use-onecc', invert: true});
```
Comment (YongseopKim) [inline src/Tests/indexCI.ts:52]: Please see `fgrep`

Comment (YongseopKim) [reply src/Tests/indexCI.ts:52]: `fgrep: '@Use-onecc'`

Comment (YongseopKim) [reply src/Tests/indexCI.ts:52]: https://github.com/Samsung/ONE-vscode/pull/423

Comment (hyunsik-yoon) [reply src/Tests/indexCI.ts:52]: related comment: https://github.com/Samsung/ONE-vscode/pull/423#issuecomment-1082815893

Comment (YongseopKim) [reply src/Tests/indexCI.ts:52]: 👍

Comment (YongseopKim) [reply src/Tests/indexCI.ts:52]: FYI, If I have an idea for a better way, I'll fix it right away.

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**